.github/actions/setup-server/action.yml

```diff
@@ -62,7 +62,7 @@ runs:
 
     - name: Install packages needed for testing
       shell: bash
-      run: python3 -m pip install nltk pytest-xdist tensorflow
```

Comment:
Remove tensorflow as a requirement for `setup-server` since we are not testing tensorflow models curently.